src/query_ctrl.ts

```diff
@@ -179,7 +216,7 @@ export class BitmovinAnalyticsDatasourceQueryCtrl extends QueryCtrl {
     return { html: filter.name, operator: { html: filter.operator || DEFAULT_OPERATOR }, filterValue: { html: filter.value || 'set filter value' } };
   }
 
-  createOrderBy(name, order) {
+  createOrderBy(name, order = null) {
     return { name, order: order || GROUPBY_DEFAULT_ORDER };
```

Comment:
If you already set the default value as a parameter default then you don't need to mess around with nullables
```suggestion
  createOrderBy(name, order = GROUPBY_DEFAULT_ORDER) {
    return { name, order: order };
```

Reply:
Nice one 👍